build_all.sh

```diff
@@ -27,17 +29,16 @@ if [[ "$target" == "linux.*" || "$target" == "macosx.*" ]]; then
   set -x
 else
   set +x
-  source "${DIR_ROOT}/sorc/machine-setup.sh"
   if [[ "${target}" == "noaacloud" ]]; then
```

Comment:
Why can't these module use/loads be placed in the "build.noaacloud.intel.lua". For example, see "build.hercules.intel.lua".

Reply:
We can restructure "build_all.sh" to make it more like others.
But have problem to load build.noaacloud.intel (the previous one).

So we have to split build.noaacloud.intel.lua into two parts,
the first part kept this name, the second part named common4noaacloud.lua.

Please take a look and see if you like the second approach.

I see other packages have similar approach.

Reply:
Changes at 90e210c look good.